src/utils/legacy.ts

```diff
@@ -3,4 +3,5 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-export const isLegacy = Number.parseInt(window.OC?.config.version.split('.')[0] ?? '32') < 32
+const version = window.OC?.config?.version || '32.0.0'
```

Comment:
Should not be needed - this is guaranteed to be available.
You just need proper global state mocks.

Reply:
Assuming properties to be present isn't great, if we really trusted, you would not have added a "32" fallback in the first place :wink: 

I think better safe than sorry :+1: